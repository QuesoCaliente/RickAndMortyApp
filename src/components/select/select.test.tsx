import { render } from '@testing-library/react';
import Select from '.';

describe('Select', () => {
  it('should render correctly with options', () => {
    const { container } = render(
      <Select
        options={[{ label: 'label', value: 'value' }]}
        onChange={() => {
          return;
        }}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
