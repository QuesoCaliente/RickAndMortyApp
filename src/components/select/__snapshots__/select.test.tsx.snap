// Vitest Snapshot v1

exports[`Select > should render correctly with options 1`] = `
<div>
  <div
    class="chakra-select__wrapper css-42b2qy"
  >
    <select
      class="chakra-select css-otbowv"
    >
      <option
        value="value"
      >
        label
      </option>
    </select>
    <div
      class="chakra-select__icon-wrapper css-162mkon"
    >
      <svg
        aria-hidden="true"
        class="chakra-select__icon"
        focusable="false"
        role="presentation"
        style="width: 1em; height: 1em; color: currentColor;"
        viewBox="0 0 24 24"
      >
        <path
          d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
          fill="currentColor"
        />
      </svg>
    </div>
  </div>
</div>
`;
